co-log-core/src/Colog/Core/IO.hs

```diff
@@ -3,18 +3,29 @@ module Colog.Core.IO
     , logStringStderr
     , logStringHandle
     , withLogStringFile
+    , liftLogIO
     ) where
 
 import Control.Monad.IO.Class (MonadIO, liftIO)
 import System.IO (Handle, IOMode( AppendMode ), hPutStrLn, stderr, withFile)
 import Colog.Core.Action (LogAction (..))
 
 
-{- | Action that prints 'String' to stdout. -}
+{- | Action that prints 'String' to stdout.
+
+>>> unLogAction logStringStdout "foo"
+foo
+
```

Comment:
Actually, empty line before `-}` is not needed. So you can safely remove it, it's redundant. If you see some empty lines in other places in documentation before `-}` you can remove them as well :+1: